libbeat/processors/add_id/add_id_test.go

```diff
@@ -61,8 +61,8 @@ func TestNonDefaultTargetField(t *testing.T) {
 	assert.NotEmpty(t, v)
 
 	v, err = newEvent.GetValue("@metadata._id")
-	assert.NoError(t, err)
-	assert.Empty(t, v)
```

Comment:
This was a bug, the right behaviour is to return an error that the key does not exist, that's what `mapstr.M` does.